content/en/blog/_posts/2017-05-00-Kubespray-Ansible-Collaborative-Kubernetes-Ops.md

```diff
@@ -3,6 +3,8 @@ title: " Kubespray Ansible Playbooks foster Collaborative Kubernetes Ops "
 date: 2017-05-19
 slug: kubespray-ansible-collaborative-kubernetes-ops
 url: /blog/2017/05/Kubespray-Ansible-Collaborative-Kubernetes-Ops
+author: >
+   Rob Hirschfeld (RackN)
 ---
 _Today’s guest post is by Rob Hirschfeld, co-founder of open infrastructure automation project, Digital Rebar and co-chair of the SIG Cluster Ops. &nbsp;_  
```

Comment:
I'd drop this note. We now just credit the person and optionally their organization.